tests/migrations/test_runner_individual.py

```diff
@@ -235,6 +235,10 @@ def perform_select_query(
     return connection.execute(full_query)
 
 
+def get_count_from_storage(storage: str, connection: ClickhousePool) -> Any:
```

Comment:
It may make sense for the return type to be an int instead of Any

Reply:
Also, I would not pass the connection into this function if possible as you can retrieve it via the storage. It'll make for a simpler API.

Reply:
So when I make it `int`, I got this error in mypy:

tests/migrations/test_runner_individual.py:239: error: Returning Any from function declared to return "int"

I think perhaps Any is the correct type then?

Reply:
You'll probably have to cast to an int so mypy won't complain

Reply:
The reason why I wrote `get_count_from_storage` the way I did was because the `table_name` and `connection` parameters are also used for `perform_select_query`. I was able to write `get_count_from_storage` in a way such that it takes in only one parameter, the storage, but we end up having to extract the table name and connection details from this storage object anyways, in migration tests. Moreover, when `get_count_from_storage` only takes in the storage object, we must also determine whether the object is ReadableTableStorage or WriteableTableStorage, and extract table name based off that, which is an extra step inside `get_count_from_storage`.

Reply:
In that case do you think I could keep it the way it is? I think another option would be to add a few other helper functions that extract the table name from any storage, and extract the connection from the storage.

Reply:
Sorry, I just realised that the `storage` argument is actually a table name and not a storage object from which we can easily retrieve the connection. It's fine as-is. You may want to rename the argument table_name or something like that instead.

Reply:
Ok, sounds good.